howso/client/base.py

```diff
@@ -9,16 +9,42 @@
     Union,
 )
 
-from howso.utilities.reaction import Reaction
 from pandas import DataFrame, Index
 
 if TYPE_CHECKING:
+    from howso.utilities.reaction import Reaction
     from .configuration import HowsoConfiguration
-    from howso.openapi.models import (
-        Cases,
-        Metrics,
-        TraineeInformation,
-    )
+
+# TODO/PR BLOCK: This map is an identity; could be changed to a list.
+# Decide if that's the desired path. Also consider what utility having
+# kwargs vs parameters defined really provides if kwargs can just be
+# used directly and parameters which aren't needed can be ignored;
+# filtering may not be necessary. Also this (and the maps below) may be
+# better placed in a constant somewhere.
+ATTRIBUTE_MAP = {
```

Comment:
Still would like resolution on this piece. As mentioned in the comment, this ATTRIBUTE_MAP seems to be an identity and if the desire is for the python attributes to match the amalgam attributes then this could probably be reduced to a list.

Reply:
We should get rid of this mapping as it no longer seems relevant. In the client method where this was utilized, we should just define these parameters explicitly.

Reply:
Addressed by: 2ba04a5